packages/file_selector/file_selector/pubspec.yaml

```diff
@@ -5,6 +5,8 @@ repository: https://github.com/flutter/plugins/tree/main/packages/file_selector/
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+file_selector%22
 version: 0.9.2
 
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
```

Comment:
please rollback these changes as they're not needed for now, we can use the package without these, until they're published. (see latest `android-package` branch)